src/Microsoft.DotNet.ImageBuilder/src/Commands/PublishImageInfoCommand.cs

```diff
@@ -93,18 +94,21 @@ private void UpdateGitRepos(string repoPath, IRepository repo)
             repo.Network.Push(branch,
                 new PushOptions
                 {
-                    CredentialsProvider = GetCredentials(Options.GitOptions.GitHubAuthOptions)
+                    CredentialsProvider = credentials
                 });
 
             Uri gitHubCommitUrl = GitHelper.GetCommitUrl(Options.GitOptions, commit.Sha);
             _loggerService.WriteMessage($"The '{Options.GitOptions.Path}' file was updated: {gitHubCommitUrl}");
         }
 
-        private static CredentialsHandler GetCredentials(GitHubAuthOptions gitHubAuthOptions) =>
-            (url, user, cred) => new UsernamePasswordCredentials
+        private async Task<CredentialsHandler> GetCredentials()
```

Comment:
Nit: async naming convention

```suggestion
        private async Task<CredentialsHandler> GetCredentialsAsync()
```

Reply:
updated PTAL